common/djangoapps/student/auth.py

```diff
@@ -88,6 +89,8 @@ def get_user_permissions(user, course_key, org=None):
         return all_perms
     if course_key and user_has_role(user, CourseInstructorRole(course_key)):
         return all_perms
+    if course_key and user_has_role(user, GlobalCourseCreatorRole(course_key.org)):
```

Comment:
nit: only org can be passed instead of course_key.org

Reply:
done